CoordgenMinimizer.cpp

```diff
@@ -37,7 +37,7 @@ static const unsigned int MAXIMUM_NUMBER_OF_SCORED_SOLUTIONS = 10000;
 static const float REJECTED_SOLUTION_SCORE = 99999999.f;
 CoordgenMinimizer::CoordgenMinimizer()
 {
-    m_maxIterations = 10000;
```

Comment:
Was this just a typo? Or were the previous changes expected to make the minimizer converge much sooner than 10000 iterations?

Reply:
It was both! Nic didn't expect that so many structures hit the max, so thought that this change wouldn't matter.

And I, in reviewing it, _read_ it as 1,000 instead of 10,000. (a "read-o", I guess)